File: web3rpc/rpc-specs/code-samples/python/eth/block/getUncleCountByBlockNumber.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockTag = "0xe8"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_uncle_count_by_block_number(blockTag)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/block/getUncleCountByBlockNumber.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const blockNumber = 119189116
    
    provider.eth.getUncleCountByBlockNumber(blockNumber, {}, (err, data, response) => {
        console.log(data);
    });
}
)()